### ISSUE TITLE
[a11y]: Fix color contrast and missing H1 elements in views

### ISSUE THREAD

**Issue by lsr-explore** [usdigitalresponse/usdr-gost#3898]
### Why is this issue important?

User's with low vision may find it more difficult to navigate the site if color contrast is low.   

Views without H1 elements impact screen reader users who use the Landmarks view of the rotor menu to understand the structure of a page.

Fix the issues identified by running axe DevTools chrome extension

## Fix Color contrast issues- use Grants branded blue
Used [Contrast Finder](https://app.contrast-finder.org/?lang=en) for suggestions for other colors
- Views
  - StandardForm.vue - Create User view - Save button
  - AgenciesView.vue - Create new Agency button
  - LoginView.vue 
  - NewTemplateView.vue - Cancel button
  - NewUploadView.vue - Submit button
  - ReportingPeriodsView.vue - multiple buttons
  - UploadsView.vue - multiple buttons
  - UservsView.vue

## Missing H1 elements
- LoginView.vue
- TemplateView.vue
- UserView.vue

### Current State

See comments in [PR #3811](https://github.com/usdigitalresponse/usdr-gost/pull/3811) - a11y(arpa): fix headings and color contrast issues  for screenshots

### Expected State

When axe DevTools is run, no color contrast or missing h1 errors should be flagged.

### Implementation Plan

Use the Grants color scheme

### Relevant Code Snippets

```shell

```
